packages/popover/src/Popover/__snapshots__/Popover.spec.js.snap

```diff
@@ -100,6 +100,38 @@ exports[`Popover Trigger Element renders with FocusLock 1`] = `
 </DocumentFragment>
 `;
 
+exports[`Popover Trigger Element renders with overlay 1`] = `
```

Comment:
Weird that this snapshot got added since you didn't add a snapshot test 🤔 

The CI also points out that this snapshot is obsolete
![image](https://user-images.githubusercontent.com/62619213/108916846-77f5ce80-75f4-11eb-90cf-30bddd7544a6.png)

Could you run a test update please

Reply:
snapshot was removed after running test:update